opennlp-tools/src/main/java/opennlp/tools/stemmer/snowball/Among.java

```diff
@@ -34,6 +34,16 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 import java.lang.reflect.Method;
 
 class Among {
+
+	public Among (String s, int substring_i, int result) {
```

Comment:
The constructor have been removed. The ```greekStemmer``` class has been modified instead to match the former ```Among``` constructor.

```hashcode``` and ```equals``` methods have been added.

Tabs have been replaced by spaces and modified to match the spaces structure that can be found on the opennlp public project.

The ```eq_s_b(CharSequence s)``` function has been modified so that it calls ```eq_s_b(int s_size, String s)``` instead.